e2e/test/svelte-jest-project/stryker.conf.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../../node_modules/@stryker-mutator/core/schema/stryker-schema.json",
+  "packageManager": "npm",
+  "testRunner": "jest",
+  "tempDirName": "stryker-tmp",
+  "concurrency": 1,
+  "logLevel": "debug",
+  "plugins": [
+    "../../../packages/jest-runner/dist/src/index.js"
```

Comment:
This should be removed. The `localDependencies` in the `e2e/package.json` file will install the `jest-runner`.